test/jit/test_string_formatting.py

```diff
@@ -136,7 +136,7 @@ def fn(arg1: List[str]) -> str:
     def test_string_interpolation_with_too_few_arguments(self):
         @torch.jit.script
         def fn(arg1: str) -> str:
-            return "%s %s in template" % arg1
+            return "{} {} in template".format(*arg1)
```

Comment:
Please keep it as is (again, to make sure that JIT frontend can parse both old and new style formatting)